tendrl/monitoring_integration/flows/update_graphite/__init__.py

```diff
@@ -0,0 +1,190 @@
+import etcd
+import os
+import datetime
+
+
+from tendrl.commons import flows
+from tendrl.commons.utils import log_utils as logger
+from tendrl.commons.utils import etcd_utils
+from tendrl.monitoring_integration.grafana import create_dashboards
+
+
+WHISPER_PATH = "/var/lib/carbon/whisper/tendrl/"
```

Comment:
Is this path the same for all versions of carbon/graphite? Can you read this path from the Graphite/Carbon conf file?

Reply:
I will retrieve the path from https://github.com/Tendrl/monitoring-integration/blob/master/etc/tendrl/monitoring-integration/graphite/carbon.conf.sample#L34, after it gets copied to the desired location